src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+public class Deadline extends Task{
+
+    protected String by;
```

Comment:
I think the string variable name "by" is not a good name for understanding. Probably change it to "deadlineDate" or "deadlineTime"?